lib/streams/transformations.dart

```diff
@@ -26,17 +25,10 @@ extension StockMapStreamTransformations on Stream<Map<int, Stock>> {
       map((stocks) => stocks[stockId]?.allTimeHigh ?? Int64(0)).distinct();
   Stream<Int64> allTimeLowStream(int stockId) =>
       map((stocks) => stocks[stockId]?.allTimeLow ?? Int64(0)).distinct();
-}
-
-extension GameStateStreamTransformations on Stream<GameStateUpdate> {
-  /// Transforms gameStateStream into a distinct game state stream
-  Stream<bool> isBankruptStream(int stockId, bool isBankrupt) => map(
-      (gameState) => gameState.gameState.stockBankruptState.stockId == stockId
-          ? gameState.gameState.stockBankruptState.isBankrupt
-          : isBankrupt).distinct();
+  Stream<bool> isBankruptStream(int stockId, bool isBankrupt) =>
+      map((stocks) => stocks[stockId]?.isBankrupt ?? isBankrupt).distinct();
```

Comment:
```suggestion
  Stream<bool> isBankruptStream(int stockId) =>
      map((stocks) => stocks[stockId]!.isBankrupt).distinct();
```
Stock map stream is ValueStream. No need of specifying initialValue for isBankrupt

Reply:
Same for givesDividends also

Reply:
okay